lib/config/presets/internal/monorepo.ts

```diff
@@ -31,6 +31,10 @@ const repoGroups = {
   'aspnet aspnetwebstack': 'https://github.com/aspnet/AspNetWebStack',
   'aspnet extensions': 'https://github.com/aspnet/Extensions',
   'aspnet-api-versioning': 'https://github.com/Microsoft/aspnet-api-versioning',
+  automapper: [
```

Comment:
```suggestion
  'automapper-dotnet': [
```
should we start unifying?